src/tools/crossgen2/ILCompiler.DependencyAnalysisFramework/DependencyAnalyzer.cs

```diff
@@ -250,42 +250,44 @@ private void ProcessMarkStack()
 
         public override void ComputeMarkedNodes()
         {
-            PerfEventSource.Log.GraphProcessingStart();
-            if (_markingCompleted)
-                return;
-
-            do
+            using (StartStopEvents graphProcessingEvents = StartStopEvents.PerfEventSource.Log.GraphProcessingEvents())
```

Comment:
Can you make these more concise? This a pretty long line for what it does:

* Since we're not actually using the `graphProcessingEvents` variable, we don't need to declare it. This can be just `using (StartStopEvents.PerfEventSource.Log.GraphProcessingEvents())`
* Since the scope of the `using` is the entire method in this case, we could use the new using declaration that was added in C# 8.0 - `using var x = ...` instead of `using (...) { }` (avoids opening the extra scope, but we need to declare a variable again)

Reply:
I personally like the first idea of `using` without the variable name. Scopes make it easier to read IMHO

Reply:
Changed to the first suggestion.